Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -4549,6 +4551,86 @@ void MainWindow::slotChannelsSpeakChannelInformationGrid(bool /*checked =false *
     slotUsersSpeakUserInformation(TT_GetMyChannelID(ttInst));
 }
 
+void MainWindow::slotChannelsSpeakChannelStatisticsGrid(bool /*checked =false */)
+{
+    slotChannelsSpeakChannelStatistics();
+}
+
+void MainWindow::slotChannelsSpeakChannelStatistics()
+{
+    QString speakList, voice, mediafile, video, desktop;
+    QVector<int> users = ui.channelsWidget->getUsersInChannel(TT_GetMyChannelID(ttInst));

```

Comment:
Shouldn't this be currently selected channel in channel-tree? Or should it only apply to own channel?

Reply:
Use ui.channelsWidget->selectedChannel always return no active users if we aren't in the specified channel, so we can probably let this apply only to the channel we are currently in